### ISSUE TITLE
[FEATURE] Placeholders in messages.yml

### ISSUE THREAD

**Issue by srnyx** [booksaw/BetterTeams#266]
**Is your feature request related to a problem? Please describe.**
Can't use `%betterTeams_tag%` in `chat.syntax` in `messages.yml`

**Describe the solution you'd like**
The ability to use Better Teams placeholders (and possibly PAPI placeholders) in `messages.yml`

**Describe alternatives you've considered**
Instead of `[TEAMNAME]`, I currently just use `[TEAM]`

**Comment by srnyx** [comment]
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.

Bump (to prevent automatic-closure)

**Comment by EarthCow** [comment]
Just tested this out and it seems to work fine. I believe this issue is already resolved. Messages.yml supports papi placeholders including betterteam's placeholders.

Related methods:
https://github.com/booksaw/BetterTeams/blob/a949af5371d6d2ce507b73eb45d02d66c1dc0e66/src/main/java/com/booksaw/betterTeams/Team.java#L1010-L1017 https://github.com/booksaw/BetterTeams/blob/a949af5371d6d2ce507b73eb45d02d66c1dc0e66/src/main/java/com/booksaw/betterTeams/message/MessageManager.java#L248-L259

**Comment by booksaw** [comment]
Yes, I have checked the changelog and it seems to have been added. I am closing this issue, please feel free to either raise a new issue or reopen this one if you encounter any problems.
